projects/plugins/jetpack/extensions/blocks/ai-assistant/editor.scss

```diff
@@ -47,6 +47,16 @@
 	.jetpack-ai-assistant__input-loader-icon {
 		color: var( --wp--preset--color--foreground );
 	}
+	
+	.jetpack-ai-assistant__input-spinner {
+		margin: 0;
+		width: 24px;
+		height: 24px;
```

Comment:
Don't know which is better. But we have the possibility to add the size from the props. Not?

Reply:
There are no props for this. On Storybook there is only one example, and it uses `style` to set `width` and `height`: https://wordpress.github.io/gutenberg/?path=/story/components-spinner--custom-size

Reply:
Looking at the code in Gutenberg repo. It has, since it's applied here: https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/spinner/index.tsx